android/src/main/java/com/thebluealliance/androidclient/fragments/tasks/UpdateUserModelSettingsTaskFragment.java

```diff
@@ -17,11 +17,13 @@
     private ModelNotificationFavoriteSettings settings;
 
     public UpdateUserModelSettingsTaskFragment() {
-
+        settings = new ModelNotificationFavoriteSettings();
     }
 
     public UpdateUserModelSettingsTaskFragment(ModelNotificationFavoriteSettings settings) {
         this.settings = settings;
+        // TODO(jerry): Call Fragment#setArguments(Bundle) to stash settings so the 0-arg
+        // constructor can call getArguments() to get them?
```

Comment:
That's the "correct" way to do it. If you use any of the fragment helper classes (or rotate the device), the fragment will be recreated with the 0-arg constructor. Probably a good idea to do